api.js

```diff
@@ -99,10 +99,27 @@ module.exports = function(_roommanager, storage) {
 			});
 			return;
 		}
-		let success = roommanager.addToQueue(req.params.name, req.body.url);
-		res.json({
-			success
-		});
+		if (req.body.url) {
+			roommanager.addToQueue(req.params.name, { url: req.body.url }).then(success => {
+				res.json({
```

Comment:
Is there a reason we are returning a json object here instead of a 200 response? If the client has logic that relies on it I am not sure it would be worth changing,

Reply:
The default status code for responses is 200. We return a json object just to return _something_. It makes it easier to debug if we are making manual requests with curl or postman.